byron/semantics/executable-spec/src/Control/State/Transition/Trace/Generator/QuickCheck.hs

```diff
@@ -42,6 +42,7 @@ import qualified Control.State.Transition as STS
 import           Control.State.Transition.Trace (Trace)
 import qualified Control.State.Transition.Trace as Trace
 
+import qualified Debug.Trace as D
```

Comment:
as discussed, maybe we should add a log/debug module that could take care of that and which could be silenced in case we don't want any output

Reply:
I happened to glance over the Haxl library today and noticed they use this [`trace_` idiom](https://github.com/facebook/Haxl/blob/1119155782233ed4e99bbf0c2a0fac9c8ee9a933/Haxl/Core/Util.hs), which might be useful for this case.

Reply:
that looks good, how about we use something like `trace_` and add an additional parameter like `info` `warning` `error` for different levels of information? One probably would not want all output in the case of looking for an error